core/modules/system/system.module

```diff
@@ -4561,3 +4561,59 @@ function system_theme_debug_enabled_warning() {
     backdrop_set_message(t('Theme debugging is enabled. See the <a href="!url">Status report</a> page for more info.', array('!url' => url('admin/reports/status'))), 'warning', FALSE);
   }
 }
+
+/**
+ * Wrapper around transliteration_get() that handles common clean up on strings.
+ *
+ * @param string $string
+ *   The string to be transliterated.
+ * @param $options
```

Comment:
Let's please add the variable type here:
```suggestion
 * @param array $options
```

Reply:
Those were preexisting. If @najmieh has time to do so, otherwise this can be a followup issue to work on core docblocks generally.